app/src/main/java/com/hammer/talkbbokki/presentation/detail/DetailScreen.kt

```diff
@@ -315,23 +347,20 @@ fun Starter() {
                 .fillMaxWidth()
         )
         Text(
-            text = "1인 1닭 가능할 것 같은 사람", style = TalkbbokkiTypography.b2_bold
+            text = starter, style = TalkbbokkiTypography.b2_bold
         )
     }
 }
 
 @Composable
 fun ShareBottom(
-    onClickShareLink: () -> Unit,
-    onClickScreenShot: () -> Unit
+    onClickShareLink: () -> Unit, onClickScreenShot: () -> Unit
 ) {
     Row(modifier = Modifier.height(62.dp)) {
-        Box(
-            modifier = Modifier
-                .fillMaxSize()
-                .weight(1f)
-                .clickable { onClickShareLink() }
-        ) {
+        Box(modifier = Modifier
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)